source/lex.tex

```diff
@@ -401,7 +401,7 @@
 preprocessing number token (one that is not a valid
 \grammarterm{integer-literal} or \grammarterm{floating-point-literal} token),
 even though a parse as three preprocessing tokens
-\tcode{0xe}, \tcode{+}, and \tcode{foo} might produce a valid expression (for example,
+\tcode{0xe}, \tcode{+}, and \tcode{foo} can produce a valid expression (for example,
 if \tcode{foo} were a macro defined as \tcode{1}). Similarly, the
```

Comment:
If we keep "can" here, then I think the parenthetical needs to say "is", not "were".

Alternatively, could we say "would be able to produce"?

Reply:
I'm happy with either. I'm a little worried that ISO will decide to declare "would" as verboten next (the upcoming revised Directives disallow use of any verbal forms not explicitly permitted, which would disallow "would"). But we can face that if and when it happens.

Reply:
fixed to "is"

We need to be able to talk about hypotheticals in proper English phrasing, that includes "could" and "were" and "would" and "might". Everything else is just stupid.
(I agree with some of the -> can changes we have been doing; they seem to be clear improvements occasionally.)